Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -133,7 +132,7 @@ ExceptionOr<RefPtr<JSC::ArrayBuffer>> CompressionStreamEncoder::compress(const u
         if (result != Z_OK && result != Z_STREAM_END && result != Z_BUF_ERROR)
```

Comment:
Nit: result is scoped to the body of this loop. I think it's clearer to declare it on the line above as it is created, rather than defining it externally without initializing.